scripts/splitControl.js

```diff
@@ -0,0 +1,172 @@
+/*global $ */
+/* exported splitControl */
+
+/*
+ * Create a splitter between two <div>s within a container.
+ * Arguments:
+ * container - ID of a <div> which contains two <div>s (herein referred
+ * to as pane1 and pane2). The splitter will be created between these.
```

Comment:
Add space before the text so it's clear that it's associated with the `container` label above it, eg:
```
 * container - ID of a <div> which contains two <div>s (herein referred
 *     to as pane1 and pane2). The splitter will be created between these.
```
Ditto other argument descriptions and returns.

Reply:
done this.